packages/ui/components/dialog/Dialog.tsx

```diff
@@ -138,7 +138,11 @@ export function DialogFooter(props: { children: ReactNode; className?: string; s
   return (
     <div className={classNames("bg-default", props.className)}>
       {props.showDivider && <hr className="border-subtle absolute right-0 w-full" />}
-      <div className={classNames("-mb-4 flex justify-end space-x-2 pt-4 rtl:space-x-reverse")}>
+      <div
+        className={classNames(
+          "flex justify-end space-x-2 pt-4 rtl:space-x-reverse",
+          props.showDivider && "-mb-4"
```

Comment:
On main branch is used this class "-mb-4" to change the padding-bottom to 16px, as this spacing should be 16px only in case when the showDivider prop is true we add this conditional